src/sick_tim_common_usb.cpp

```diff
@@ -264,6 +264,7 @@ int SickTimCommonUsb::sendSOPASCommand(const char* request, std::vector<unsigned
 {
   if (device_handle_ == NULL) {
     ROS_ERROR("LIBUSB - device not open");
+    sleep(1);
```

Comment:
Moved the sleep:

https://github.com/chadrockey/sick_tim/commit/f07fa2092435e19095d60ed6400ff64688f5df36